session/balance/provider/balance_tracker_test.go

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package provider
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/mysteriumnetwork/node/money"
+)
+
+func Test_BalanceTracker(t *testing.T) {
+	var initialBalance uint64 = 100
+	mockTime := time.Second
+	mockMoney := money.Money{
+		Amount:   1,
+		Currency: money.CURRENCY_MYST,
+	}
+	mtk := &mockTimeKeeper{elapsed: mockTime}
+	mac := &mockAmountCalculator{toReturn: mockMoney}
+	tracker := NewBalanceTracker(mtk, mac, initialBalance)
+	balance := tracker.GetBalance()
+
+	balanceAfter := initialBalance - mockMoney.Amount
```

Comment:
Static values in asserts liek `1` and `2` would make test more readable, I would say